nav2_map_server/include/nav2_map_server/occ_grid_server.hpp

```diff
@@ -12,35 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef NAV2_MAP_SERVER__MAP_REPS__OCC_GRID_SERVER_HPP_
-#define NAV2_MAP_SERVER__MAP_REPS__OCC_GRID_SERVER_HPP_
+#ifndef NAV2_MAP_SERVER__OCC_GRID_SERVER_HPP_
+#define NAV2_MAP_SERVER__OCC_GRID_SERVER_HPP_
 
 #include <memory>
 #include <string>
 
+#include "rclcpp/rclcpp.hpp"
 #include "nav_msgs/msg/occupancy_grid.hpp"
 #include "nav_msgs/msg/map_meta_data.hpp"
 #include "nav_msgs/srv/get_map.hpp"
-#include "rclcpp/rclcpp.hpp"
-#include "nav2_map_server/base_map_server.hpp"
+#include "nav2_map_server/map_server.hpp"
 
 namespace nav2_map_server
 {
 
-class OccGridServer : public BaseMapServer
+class OccGridServer : public MapServer
 {
 public:
-  explicit OccGridServer(rclcpp::Node::SharedPtr node)
-  : node_(node) {}
+  explicit OccGridServer(rclcpp::Node::SharedPtr node);
   OccGridServer(rclcpp::Node::SharedPtr node, std::string file_name);
   OccGridServer() {}
-  ~OccGridServer() {}
 
-  void LoadMapInfoFromFile(const std::string & file_name);
-  void LoadMapFromFile(const std::string & map_name);
-  void PublishMap();
-  void SetMap();
-  void ConnectROS();
+  void loadMapInfoFromFile(const std::string & file_name);
```

Comment:
This function can be modified to simply grab the parameters from the server (intiialized via launch in the yaml) and hence not require a filename to parse the yaml file directly.

Reply:
Yes, will do.